requirements/main.txt

```diff
@@ -1,17 +1,18 @@
 diff_match_patch
 editdistance
+ffmpeg
 hydra-core
 joblib
 librosa>=0.10.0 # specify >=0.10.0 so that librosa.get_duration(path=...) will work
 numpy
 omegaconf
 pandas
+rarfile
 regex
 sox
 tqdm
-wget
-ffmpeg
-rarfile
 webvtt-py
+wget
+
 # for some processers, additionally https://github.com/NVIDIA/NeMo is required
-# for some processers, additionally nemo_text_processingis required
```

Comment:
put this line back, don't delete?

Reply:
and fix typo: "# for some processers, additionally nemo_text_processing is required"